src/Fantomas.Core.Tests/UnionTests.fs

```diff
@@ -879,6 +879,25 @@ type Foo = // comment
         1
 """
 
+[<Test>]
+let ``comment after variants with modules`` () =
```

Comment:
Could you add the issue number, please?
`comment after variants with modules, 2103`

Reply:
Actually, that test name isn't really good.
This has nothing to do with modules.
Maybe go for `restore trivia after long identifier in SynField`.
I'll update the original issue title as well.

Reply:
I have updated this PR following your suggestions :)